app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,6 @@
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:roundIcon="@mipmap/ic_launcher_round"
```

Comment:
Why to remove from manifest?

Reply:
Beacuse App icon is one of Rectangular type and no need of round icon. Round icon is not included.

Reply:
Brilliant, would be great if you could attach screenshot to PR.

Reply:
![Screenshot_2019-11-09-00-06-28-824_com miui home](https://user-images.githubusercontent.com/35262074/68502204-e25ef000-0285-11ea-8155-b592f0c0406b.png)

Reply:
great ask @annabauza its always good to have visuals when the PR changes something in the UI or other visuals :)